apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: rate-limits
```

Comment:
should we rename this file "rate-limits.mdx"? Not sure if that matters for the URL?

Reply:
Doesn't matter for the URL but let's do it for consistency.